src/components/TopBar/TopBar.jsx

```diff
@@ -1,64 +1,136 @@
-import React from 'react';
-// import { StaticJsonRpcProvider, Web3Provider } from "@ethersproject/providers";
-import { useSelector, useDispatch } from 'react-redux';
-// import Web3Modal from "web3modal";
-import { shorten } from '../../helpers';
-import ThemeSwitcher from "../ThemeSwitch/ThemeSwitch";
+import React, { useState } from "react";
 import { Flex } from "rimble-ui";
-import useMediaQuery from "@material-ui/core/useMediaQuery";
+import InfoIcon from "@material-ui/icons/Info";
+import ThemeSwitcher from "../ThemeSwitch/ThemeSwitch";
 import "./topbar.scss";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import { addresses } from "../../constants";
 
-function TopBar({ web3Modal, loadWeb3Modal, logoutOfWeb3Modal, address, theme, toggleTheme}) {
-	const isVerySmallScreen = useMediaQuery("(max-width: 649px)");
-	const isUltraSmallScreen = useMediaQuery("(max-width: 495px)");
+const SOHM_ADDRESS = addresses[1].SOHM_ADDRESS;
```

Comment:
You should pass `networkID` to TopBar instead of hardcoding. See https://github.com/OlympusDAO/olympusdao/blob/03b58ba63b802bdac8d1b5710c004ef0c556242d/src/helpers/index.js#L20

Reply:
I tried, but couldn't find a reducer that is providing it, where is it?